### PR TITLE
Improves Spoon's test

### PR DESCRIPTION
**Pull request by GerardPaligot**

### CODE REVIEW

File: src/test/java/spoon/test/targeted/TargetedExpressionTest.java [hunk 1]
```diff
@@ -265,7 +252,7 @@ public void testStaticTargetsOfFieldAccessNoClasspath() throws Exception {
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(2));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("Foo.this.i"), elements.get(3));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(4));
-		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess).result("Foo.staticField"), elements.get(5));
+		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess.toString()).result("Foo.staticField"), elements.get(5));
```
Comment (monperrus) [inline src/test/java/spoon/test/targeted/TargetedExpressionTest.java]: required?

File: src/test/java/spoon/test/targeted/TargetedExpressionTest.java [hunk 2]
```diff
@@ -456,18 +430,16 @@ public void testClassDeclaredInALambda() throws Exception {
 		final List<CtFieldAccess<?>> elements = type.getElements(new TypeFilter<>(CtFieldAccess.class));
 		assertEquals(3, elements.size());
 
-		final CtThisAccess<Object> exepectedThisAccess = type.getFactory().Core().createThisAccess();
-
 		final CtTypeReference<Object> firstExpected = type.getFactory().Type().createReference("spoon.test.targeted.testclasses.Tapas$1$InnerSubscriber");
-		exepectedThisAccess.setType(firstExpected);
+		CtThisAccess<Object> exepectedThisAccess = type.getFactory().Code().createThisAccess(firstExpected);
```
Comment (monperrus) [inline src/test/java/spoon/test/targeted/TargetedExpressionTest.java]: typo in variable name

**Comment by monperrus** [comment]
looks good

File: src/test/java/spoon/test/targeted/TargetedExpressionTest.java (hunk 1 above)
Comment (GerardPaligot) [inline src/test/java/spoon/test/targeted/TargetedExpressionTest.java]: The target is a `CtCodeSnippetExpression`. For now, it isn't necessary but it'll be with the equals refactoring.

Comment (monperrus) [inline src/test/java/spoon/test/targeted/TargetedExpressionTest.java]: do you mean that after the refactoring, a CtSnippet will not be equal
with an object that represents the same code? but currently it works?
(it makes sense)

Comment (GerardPaligot) [inline src/test/java/spoon/test/targeted/TargetedExpressionTest.java]: Probably.

**Pull request merged**